sfge/src/main/java/com/salesforce/rules/ops/visitor/LoopDetectionVisitor.java

```diff
@@ -105,8 +146,27 @@ private void createOverridableLoopExclusion(
         }
     }
 
+    // necessary to limit the _afterLoopVisit method to only Method calls and Soql calls
```

Comment:
This comment feels weirdly placed. Can it be converted into a JDoc-style comment somehow?